featuretools/tests/primitive_tests/test_agg_feats.py

```diff
@@ -124,6 +125,16 @@ def test_check_input_types(es):
     assert mean._check_input_types()
 
 
+def test_mean_nan():
+    mean_func = Mean().get_function()
+    array = np.array([5, 5, 5, 5, 5])
+    assert mean_func(array) == 5
+    array = np.array([5, np.nan, np.nan, np.nan, np.nan, 10])
```

Comment:
you should add a test case where there are nans in the list, but `ignore_nans=False`

Reply:
Fixed.